chapters/handling-data.tex

```diff
@@ -1,57 +1,42 @@
-	We also expect teams will report what they have and have not done
-	in order to provide objective measures of a research product's performance in both.
-	Otherwise, reputation is the primary signal for the quality of evidence, and two failures may occur:
-	low-quality studies from reputable sources may be used as evidence when in fact they don't warrant it,
-	and high-quality studies from sources without an international reputation may be ignored.
-	Both these outcomes reduce the quality of evidence overall.
-	Even more importantly, the only way to determine credibility without transparency
-  is to judge research solely based on where it is done and by whom,
-  which concentrates credibility at better-known international institutions and global universities,
-  at the expense of quality research done by people and organizations directly involved in and affected by it.
-	Simple transparency standards mean that it is easier to judge research quality,
-	and making high-quality research identifiable also increases its impact.
-	This section provides some basic guidelines and resources
-	for using field data ethically and responsibly to publish research findings.
+	As the range and importance of the policy-relevant questions asked by development researchers grow,
+	so too does the (rightful) scrutiny under which methods and results are placed.
+	It is useful to think of research as a public service,
+	one that requires you to be accountable to both research participants and research consumers.
+	On the research participant side,
+	it is essential to respect individual privacy and ensure data security.
+	Researchers look deeply into real people's personal lives, financial conditions, and other sensitive subjects.
+	Respecting the respondents' right to privacy,
+	by intelligently assessing and proactively averting risks they might face,
+	is a core tenet of research ethics.
+	On the consumer side, it is important to protect confidence in development research
+	by following modern practices for transparency and reproducibility.
+
+  Across the social sciences, the open science movement has been fueled by discoveries of low-quality research practices,
+	data and code that are inaccessible to the public, analytical errors in major research papers,
+	and in some cases even outright fraud. While the development research community has not yet
+	experienced any major scandals, it has become clear that there are necessary incremental improvements
+	in the way that code and data are handled as part of research.
+	Neither privacy nor transparency is an all-or-nothing objective:
+	the most important thing is to report the transparency and privacy measures you have taken
+  and always strive to to the best that you are capable of with current technology.
+	In this chapter, we outline a set of practices that help to ensure
+	research participants are appropriately protected and
+	research consumers can be confident in the conclusions reached.
```

Comment:
I am happy to be overturned, but would it be nice to mention already here that this chapter "outlines a set of practices" and later chapters provide more hands-on guides to those practices? For example encryption, de-identification etc.

Reply:
Added this line in 10abdabf90ecdae78a82b41d0d0289282716a4cc